### ISSUE TITLE
regex warning generated for > CPython 3.6: Flags not at the start of the expression

### ISSUE THREAD

**Issue by Wim-De-Clercq** [Pylons/pyramid_debugtoolbar#347]
Since CPython 3.6 a warning will be displayed when flags inside of a regex pattern are not at the beginning of the pattern: https://github.com/python/cpython/blob/3.6/Lib/sre_parse.py#L736

The [tbtools.py](https://github.com/Pylons/pyramid_debugtoolbar/blob/4.5/src/pyramid_debugtoolbar/tbtools.py#L35) file has such a pattern where the flags are at the end.

Example warning:
```
.venv/lib/python3.6/site-packages/pyramid_debugtoolbar/tbtools.py:35: DeprecationWarning: Flags not at the start of the expression b'^(.*?)$(?m)'
    _line_re = re.compile(br'^(.*?)$(?m)')
```

**Comment by tdamsma** [comment]
Came here looking to address this, but see it is already solved and merged in #348. When is the next release planned?

**Comment by merwok** [comment]
This was released in 4.5.1!

**Comment by mmerickel** [comment]
Yep fixed. Thanks for the reminder.
